File: java/Gate-Two/Level 4/Task5.java
public class Task5{ 
	static boolean numIsPrime(int num) {
		if (num < 2) return false;
    		for (int index = 2; index < num; index++) {
       			 if (num % index == 0) return false;
    }
    	return true;
}
public static void main(String...args) {
    System.out.println(numIsPrime(11));   
    System.out.println(numIsPrime(15));  
}
}

File: javaScript/Gate-Two/Task5.js
function boolean numIsPrime(let num) {
		if (num < 2) return false;
    		for (let index = 2; index < num; index++) {
       			 if (num % index == 0) return false;
    }
    	return true;
}
console.log(numIsPrime(11));   
console.log(numIsPrime(15));  
